### ISSUE TITLE
Sync client returns async resources

### ISSUE THREAD

**Issue by mdrost** [php-api-clients/github#15]
You should not create sync client from async client because returned resources are async and their refresh function returns promises instead of resource itself.

**Comment by WyriHaximus** [comment]
Yeah I'm aware of that, was planning to look into that today or tomorrow
